proposals/load_geojson.json

```
@@ -0,0 +1,53 @@
+{
+    "id": "load_geojson",
+    "summary": "Converts GeoJSON into a vector data cube",
+    "description": "Converts GeoJSON data as defined by [RFC 7946](https://www.rfc-editor.org/rfc/rfc7946.html) into a vector data cube. Feature properties are preserved.",
+    "categories": [
+        "import",
+        "vector"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A GeoJSON object to inline into the process. The GeoJSON type `GeometryCollection` is not supported. Each geometry in the GeoJSON data results in a dimension label in the `geometries` dimension.",
```

> A GeoJSON object to inline into the process.

Statement looks a bit weird to me: the object is not to be inlined into the process, it is inlined in the process graph and is to be converted to a vector cube.

```suggestion
            "description": "A GeoJSON object to convert to a vector cube. The GeoJSON type `GeometryCollection` is not supported. Each geometry in the GeoJSON data results in a dimension label in the `geometries` dimension.",
```

---

```suggestion
            "description": "A GeoJSON object to convert to a vector data cube. The GeoJSON type `GeometryCollection` is not supported. Each geometry in the GeoJSON data results in a dimension label in the `geometries` dimension.",
```

---

Updated.